<?xml version="1.0"?>
<launch>
    <!-- Run filtering -->
    <node name="image_formater_node" pkg="ros_ml" type="image_formater_node" output="screen">
        <!-- Frame rate -->
        <param name="frame_rate" value="1.0" />

        <!-- Image topic -->
        <param name="image_topic" value="/econ_camera/image_raw/compressed" />

        <!-- Saving directory -->
        <param name="save_directory" value="images/" />

        <!-- Start index -->
        <param name="start_index" value="0" />
    </node>
</launch>
